content/docs/command-reference/commit.md

```diff
@@ -34,9 +34,11 @@ scenarios are further detailed below.
 
 - It's always possible to manually execute the source code used in a stage
   without DVC (outputs must be unprotected or removed first in certain cases,
-  see `dvc unprotect`). Once a desirable result is reached, use `dvc add` or
-  `dvc commit` as appropriate to update the `dvc.lock` or `.dvc` files and store
-  changed data to the cache.
+  see `dvc unprotect`). We can also
+  [add outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  without executing it again. Once a desirable result is reached, use `dvc add`
+  or `dvc commit` as appropriate to update the `dvc.lock` or `.dvc` files and
+  store changed data to the cache.
```

Comment:
Let's use 2 bullets. The existing one is having run a command without DVC and later wrapping the results with a dvc.yaml file. BTW this one should be the last bullet and could use some improvement in it's explanation e.g. it's not necessarily about source code, it could be a binary, etc. (maybe use the term "terminal command")...

The new one is about having run a stage with DVC (either writing dvc.yaml manually and using `repro`, or with `run`) but later noticing dvc.yaml is incomplete, missing one+ output(s) — which already exist in the workspace, so no need to re-execute the command here either.

So as you can see there's lots of subtleties here and we need to express everything in the docs as clearly and briefly as possible (difficult balance).